test/Microsoft.AspNetCore.SignalR.Tests.Utils/Microsoft.AspNetCore.SignalR.Tests.Utils.csproj

```diff
@@ -5,10 +5,6 @@
     <RootNamespace>Microsoft.AspNetCore.SignalR.Tests</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpPackageVersion)" />
```

Comment:
Can it also be removed from Microsoft.AspNetCore.SignalR.Tests.csproj?

Reply:
Maybe, we do have things that test the dynamic hubs so it may be required.

Reply:
Yea, I'm going to leave that one in https://github.com/aspnet/SignalR/blob/8d4945df8870e74e1ffa722f28215dcd077da665/test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTestUtils/Hubs.cs#L171